Proj4000: Invalid INI file (#51)
dotnet-project-file-analyzers/dotnet-project-file-analyzers.github.io: rules/Proj4002.md

# Proj4002: Invalid INI key-value pair
An [INI file](https://en.wikipedia.org/wiki/INI_file) is a plain text file with
a structure and syntax comprising key–value pairs organized in sections.

> Key-value pairs can be both seperated by `=` and `:`.

Reviewer: ```suggestion
Key-value pairs can either be separated by `=` or `:`.
```
